data/sql/updates/pending_db_world/rev_1616472392022033200.sql

```diff
@@ -0,0 +1,477 @@
+INSERT INTO `creature_text_locale` (`CreatureID`, `GroupID`, `ID`, `Locale`, `Text`) VALUES
+(22431,2,1,'esMX','¡El poder de la luz te obliga! ¡De vuelta a tu pozo!'),
+(22431,2,2,'esMX','¡Purifícate con Luz, humano! No dejes que la corrupción demoníaca te abrume.'),
+(22431,2,3,'esMX','La luz es mi guía... ¡es mi sustento!'),
+(22431,2,4,'esMX','Yo... ¡no debo... vacilar!'),
+(22431,2,5,'esMX','¡En nombre de la Luz! ¡Es la Luz la que te manda! ¡Es la Luz que te arrojó a las profundidades de la oscuridad!');
+
+
+DELETE FROM `creature_text_locale` WHERE `CreatureID` = 22432 AND `locale` IN ('esMX');
+INSERT INTO `creature_text_locale` (`CreatureID`, `GroupID`, `ID`, `Locale`, `Text`) VALUES
+(22432,1,1,'esMX','Danos tiempo... Deja que el hombre muera... No soy nadie... No soy nadie... Teme al anacoreta... Teme al anacoreta... Barada... Barada.'),
+(22432,1,2,'esMX','¡Esto es infructuoso, draenei! Tú y tu pequeño ayudante no pueden arrebatar el control de este patético humano.'),
+(22432,1,3,'esMX','Haré pedazos su alma y lo asaré lentamente sobre fuego demoníaco.'),
+(22432,1,4,'esMX','¡Todo está perdido, Anacoreta! Abandona la esperanza que queda.'),
+(22432,1,5,'esMX','¡Veo a tus antepasados, Anacoreta! Se retuercen y gritan en la oscuridad... ¡están con nosotros!');
+
+
+DELETE FROM `creature_text_locale` WHERE `CreatureID` = 21504 AND `locale` IN ('esES');
+INSERT INTO `creature_text_locale` (`CreatureID`, `GroupID`, `ID`, `Locale`, `Text`) VALUES
+(21504,20,0,'esES','¿Y qué crees que estás haciendo? ¿Te atreves a interferir con el experimento de mi maestro?'),
+(21504,21,0,'esES','¿Te gusta lo que hemos hecho aquí? Quizás arrojemos estos cristales desde el cielo por todo Terrallende.'),
+(21504,22,0,'esES','Me aburro en tu intento de curar la tierra y sofocar las energías que convocan y vuelven locos a los colosos. ¡Goliathon rey de los colosos, el príncipe Kael’thas y yo exigimos que defiendas el cristal!'),
+(21504,23,0,'esES','Nos volveremos a encontrar, pronto.');
+
+
+DELETE FROM `creature_text_locale` WHERE `CreatureID` = 21504 AND `locale` IN ('esMX');
+INSERT INTO `creature_text_locale` (`CreatureID`, `GroupID`, `ID`, `Locale`, `Text`) VALUES
+(21504,20,0,'esMX','¿Y qué crees que estás haciendo? ¿Te atreves a interferir con el experimento de mi maestro?'),
+(21504,21,0,'esMX','¿Te gusta lo que hemos hecho aquí? Quizás arrojemos estos cristales desde el cielo por todo Terrallende.'),
+(21504,22,0,'esMX','Me aburro en tu intento de curar la tierra y sofocar las energías que convocan y vuelven locos a los colosos. ¡Goliathon rey de los colosos, el príncipe Kael’thas y yo exigimos que defiendas el cristal!'),
+(21504,23,0,'esMX','Nos volveremos a encontrar, pronto.');
+
+UPDATE `quest_template_locale` SET `ObjectiveText1` = "Jules vengado" WHERE `ID` = 10909 AND `locale` IN ('esES');
+UPDATE `quest_template_locale` SET `ObjectiveText1` = "Jules vengado" WHERE `ID` = 10909 AND `locale` IN ('esMX');
```

Comment:
@MathematicalWTF thanks for your contribution. You can combine this kind of `UPDATE` queries using:

```suggestion
UPDATE `quest_template_locale` SET `ObjectiveText1` = "Jules vengado" WHERE `ID` = 10909 AND `locale` IN ('esES', 'esMX');
```